### PR TITLE
Clearly distinguish between http and jmx ports

### PR DESCRIPTION
**Pull request by gkumar7**
A minor fix to README to ensure the same port is not being monitored and used to serve the metrics.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ Agent is thus strongly encouraged.
 To run as a javaagent [download the jar](https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.2.0/jmx_prometheus_javaagent-0.2.0.jar) and run:
 
 ```
-java -javaagent:./jmx_prometheus_javaagent-0.2.0.jar=1234:config.yaml -jar yourJar.jar
+# Serve metrics on port 8080
```
Comment (brian-brazil) [inline README.md]: This line is a bit distracting, what we usually do is include a link to the /metrics in the prose.

Comment (gkumar7) [reply README.md]: Sure, will update

File: README.md [hunk 2]
```diff
@@ -16,8 +16,9 @@ Agent is thus strongly encouraged.
 To run as a javaagent [download the jar](https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.2.0/jmx_prometheus_javaagent-0.2.0.jar) and run:
 
 ```
-java -javaagent:./jmx_prometheus_javaagent-0.2.0.jar=1234:config.yaml -jar yourJar.jar
+java -javaagent:./jmx_prometheus_javaagent-0.2.0.jar=8080:config.yaml -jar yourJar.jar
 ```
+Metrics will now be accessible at http://localhost:8080
```
Comment (brian-brazil) [inline README.md]: Don't forget the /metrics

Comment (gkumar7) [reply README.md]: Both should work, agree? 

https://github.com/prometheus/client_java/blob/master/simpleclient_httpserver/src/main/java/io/prometheus/client/exporter/HTTPServer.java#L147-L148

Regardless, will update.

Comment (brian-brazil) [reply README.md]: They both work, but that's more that I haven't gotten around to adding a home page.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**